articles/budougumi0617.re

```diff
@@ -0,0 +1,14 @@
+= 動作確認用
+
+== はじめに
+
+freee株式会社でバックエンドエンジニアをしている@budougumi0617@<fn>{bd617_twitter}です。
+
+//footnote[bd617_twitter][@<href>{https://twitter.com/budougumi0617}]
+
+=== 本章の目的・ゴール
+このファイルはreviewdogの動作確認用です！
```

Comment:
**[textlint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub>
Disallow to use "！". (ja-technical-writing/no-exclamation-question-mark)